CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet
+### Fixed
+- call.HeaderNames() now creates specifies a capacity when creating a slice, 
```

Comment:
`now creates specifies` -> `now specifies`

Reply:
Thanks for finding this; I've updated the diff to fix the typo.